lib/cqm/converter/utils.rb

```diff
@@ -38,19 +38,9 @@ def self.gather_qdm_model_attrs
           qdm_model_attrs[model.to_s].concat(QDM.const_get(model).embedded_relations.keys.map! { |a| a.camelize(:lower) })
         end
       end
-      # TODO: These fields are currently not supported. See:
+      # TODO: This fields is currently not supported. See:
```

Comment:
Technically we should probably singularize `fields`. Otherwise it looks good.

Reply:
Doh! Good catch!